### PR TITLE
[4.13] Configure 4.13 mirroring lists

### PR DESCRIPTION
**Pull request by jmguzik**
Signed-off-by: Jakub Guzik <jguzik@redhat.com>

### CODE REVIEW

File: core-services/image-mirroring/openshift/mapping_origin_4_13 [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+registry.ci.openshift.org/origin/4.13:libvirt-installer quay.io/openshift/origin-libvirt-installer:4.13 quay.io/openshift/origin-libvirt-installer:4.13.0
+registry.ci.openshift.org/origin/4.13:machine-os-content quay.io/openshift/origin-machine-os-content:4.13 quay.io/openshift/origin-machine-os-content:4.13.0
+registry.ci.openshift.org/origin/4.13:machine-os-images quay.io/openshift/origin-machine-os-images:4.13 quay.io/openshift/origin-machine-os-images:4.13.0
+registry.ci.openshift.org/origin/4.13:openstack-installer quay.io/openshift/origin-openstack-installer:4.13 quay.io/openshift/origin-openstack-installer:4.13.0
+registry.ci.openshift.org/origin/4.13:operator-marketplace quay.io/openshift/origin-operator-marketplace:4.13 quay.io/openshift/origin-operator-marketplace:4.13.0
+registry.ci.openshift.org/origin/4.13:ovirt-installer quay.io/openshift/origin-ovirt-installer:4.13 quay.io/openshift/origin-ovirt-installer:4.13.0
+registry.ci.openshift.org/origin/4.13:upi-installer quay.io/openshift/origin-upi-installer:4.13 quay.io/openshift/origin-upi-installer:4.13.0
```
Comment (bparees) [inline core-services/image-mirroring/openshift/mapping_origin_4_13:17]: why is this list so much shorter than the 4.12 one?

Comment (jmguzik) [reply core-services/image-mirroring/openshift/mapping_origin_4_13:17]: I think it's related to the moment of time we are currently in. Not everything is ready to have a complete list, yet it's not the purpose of this PR (which is just the last step of our post-code freeze activities). More context: https://github.com/openshift/release/pull/26644#issuecomment-1055547549

Comment (bparees) [reply core-services/image-mirroring/openshift/mapping_origin_4_13:17]: i guess it doesn't matter for now as long as the list will get regenerated when we really start 4.13.

**Pull request merged**